frontend/local/src/modules/ehr/encounter.js

```diff
@@ -154,12 +184,24 @@ export default dispatch =>
             unit: "/min",
             formPath: "vitalSigns.heart_rate.value"
         },
+        {
+            type: "dateTime",
+            ehrPath:
+                "/content[openEHR-EHR-COMPOSITION.encounter.v1]/context/other_context/items[openEHR-EHR-OBSERVATION.pulse.v1]/context/start_time",
+            formPath: "vitalSigns.pulse.timestamp"
+        },
 
         // oxygen saturation (0 - 100)
         {
             type: "value",
             ehrPath:
                 "/content[openEHR-EHR-COMPOSITION.encounter.v1]/context/other_context/items[openEHR-EHR-OBSERVATION.pulse_oximetry.v1]/data[at0001]/events[at0002]/data[at0003]/items[at0006]",
             formPath: "vitalSigns.oxygen_saturation.value"
+        },
+        {
+            type: "dateTime",
+            ehrPath:
+                "/content[openEHR-EHR-COMPOSITION.encounter.v1]/context/other_context/items[openEHR-EHR-OBSERVATION.pulse_oximetry.v1]/context/start_time",
+            formPath: "vitalSigns.oxygen_saturation.timestamp"
```

Comment:
the `events` requries the `:0` suffix and the timestamp should be stored as under `/time`